ui/src/app/subnets/views/VLANDetails/ConfigureDHCP/ConfigureDHCPFields/ConfigureDHCPFields.tsx

```diff
@@ -48,12 +48,6 @@ const ConfigureDHCPFields = ({ vlan }: Props): JSX.Element => {
         <FormikField
           label="MAAS provides DHCP"
           name="enableDHCP"
-          onChange={async (e: ChangeEvent) => {
-            await handleChange(e);
-            setFieldValue("primaryRack", "");
-            setFieldValue("relayVLAN", "");
-            setFieldValue("secondaryRack", "");
-          }}
```

Comment:
I could have updated this to act differently depending on whether `e.target.value` is true or false, but the parent `onSubmit()` handler already handles how to shape the API so it's actually unnecessary to do anything here.